Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -93,6 +96,7 @@ struct FPTIBatchData: Codable {
         }
 
         enum CodingKeys: String, CodingKey {
+            case appSwitchURL = "appSwitchUrl"
```

Comment:
`appSwitchUrl` is not an approved tag in FPTI. We will want to use `url` here instead.

Reply:
Got it, thanks for the feedback, updated: [714dd27](https://github.com/braintree/braintree_ios/pull/1435/commits/714dd27c44918283e04160d130bec7687a3d49bd)